### PR TITLE
link sorted

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
I created the add new and contact page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,22 +10,44 @@
...
+                <li>Our address: Streetname 22, 84839 City, Country.</li>
+            </ul>
+        </section>
+
+        <form id="form" class="form1">
             <h3>Add a new book</h3>
             <p> <input type="text" placeholder="Title" id="title"> </p>
             <p> <input type="text" placeholder="Author" id="author"> </p>
             <button type="submit" id="add">Add</button>
         </form>
```
Comment (AbelG101) [inline index.html:48]: - [ ] Ouch! this is really close but at a given moment only one of the sections should be displayed and you seem to be displaying all of the sections. check the video below 👇
    firefox_VMYJST7017
- As you can see when the page loads all the sections are displayed so please work on that ✔
- **HINT**: You can give the "Contact" & "Add book" sections a default value of `display: none;`.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi Team👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- UI looking exactly like wireframe ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @Simpleshaikh1 and @zbahati, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job resolving the bug from the previous review :rocket: 


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] There is another bug we need to resolve, once the page is loaded, there is an empty page, once data is inserted, we still see an empty page even after adding data. Even after page refresh we still see an empty page unless we click on the list in the navbar before seeing the data. Let us kindly fix it

 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Simpleshaikh1 @zbahati, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**